### PR TITLE
Changing Vanity URL Code

### PR DESCRIPTION
**Pull request by muratvastark**
I wanted to change my server's special invitation, but it was not available, I wanted to add :3

### CODE REVIEW

**Comment by ImRodry** [comment]
There's a reason this isn't supported and that is that Discord does not support this. This code is simply wrong, you don't follow the PR template nor the commitlint style, and you also have formatting and documentation issues

**Comment by DTrombett** [comment]
By the way, usually PRs to v13 branch are not accepted

**Comment by muratvastark** [comment]
> There's a reason this isn't supported and that is that Discord does not support this. This code is simply wrong, you don't follow the PR template nor the commitlint style, and you also have formatting and documentation issues

😢

**Comment by muratvastark** [comment]
> Did you even test this?

yeah I tested it and it works
image

**Comment by ImRodry** [comment]
> > Did you even test this?
> 
> yeah I tested it and it works image

well in that case this needs to be documented on the Discord API docs, which it isn't ATM. I suggest opening a PR there first and closing this, as PRs to v13 will not be accepted

File: src/structures/Guild.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ let deprecationEmittedForSetChannelPositions = false;
 let deprecationEmittedForSetRolePositions = false;
 let deprecationEmittedForDeleted = false;
 
+
```
Comment (xHyroM) [inline src/structures/Guild.js:43]: Empty line?

**Comment by wasdennnoch** [comment]
> well in that case this needs to be documented on the Discord API docs, which it isn't ATM. I suggest opening a PR there first and closing this, as PRs to v13 will not be accepted

This has been denied in the past - although I can't say if their stance has changed in the past 4 years.

https://github.com/discord/discord-api-docs/issues/519

**Comment by xHyroM** [comment]
Can you fix eslint?

**Pull request closed without merging**